contrib/NeonVM/string.c

```diff
@@ -0,0 +1,146 @@
+#ifdef _MSC_VER
+#pragma warning (push)
+#pragma warning (disable: 4001)
```

Comment:
What is warning 4001? This stuff should be commented with what it actually does.

Reply:
Commented.